import "game_commands.proto";
message Command_UndoDraw {
    extend GameCommand {
        optional Command_UndoDraw ext = 1007;
    }
}
